src/xiaomi_parser.py

```diff
@@ -374,11 +367,15 @@ def build_patterns(models):
         if "ir_zip_key_r" in model:
             # Decrypt IR code
             ir_code = process_xiaomi_shit(model["ir_zip_key_r"])
+            button_name = model.get("id", None)
```

Comment:
- None is already a default val
- Understood the need to keep the name of the button 
- Will rename id to something more explicit 'key_name'
- build_patterns was made to build patterns from models in json brands not models in models/ dir.
Will do a separated func for that